src/options.c

```diff
@@ -13,13 +13,15 @@
 #include <strings.h>
 #endif
 #include <stdlib.h>
+#ifdef HAVE_UNISTD_H
 #include <unistd.h>
```

Comment:
Out of curiosity, what is this used for in this file?

Reply:
Hm, removing it is OK in linux builds. Will test a few others and remove if no issues.

Reply:
OK, removed it along with some other includes as of 6ac2b961c59f4c929d24ea3ea9919add40f78937

Reply:
`getopt_long` is supposed to be in unistd.h.

https://linux.die.net/man/3/getopt_long

Reply:
No: only getopt() itself is. getopt_long and getopt_long_only are in
getotpt.h, as documented on that page. (unless I'm missing things.)

Reply:
You are correct, I missed the second include.